### ISSUE TITLE
Predict model for evalute is Zero ?

### ISSUE THREAD

**Issue by NguyenThaiHoc1** [Neerajj9/CSRNet-keras#31]
I have a problems, After I training but model predict=0.0, I don't know why,It's really tough to me.
I still try to change learning rate. But the problem don't handle
Please help me

**Comment by Tangxy666** [comment]
I have the same problem. Have you found the reason？
